### ISSUE TITLE
`js` library currently not working

### ISSUE THREAD

**Issue by mk270** [ursalang/ursa#9]
https://github.com/ursalang/ursa/blob/main/test/use-fs.ursa

suggests i can do
```
use js.fs
```
but
```
> use js.fs
Line 1, col 5:
> 1 | use js.fs
          ^~

Undefined symbol js
```

**Comment by rrthomas** [comment]
Apologies, this test is currently not run, and indeed does not work; the error message is correct!

I'll leave this issue open in case anyone else stumbles on this.

**Comment by rrthomas** [comment]
This was fixed in 0.1.16.
